New icon: helix (original, plain, line)

Shyynek opened the pull request:
## Double check these details before you open a PR

<!-- Tick the checkboxes to ensure you've done everything correctly -->
- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [x] A new object is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

<!-- Refer to the [contributing](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) guidelines for more details. -->

**This PR closes NONE**
<!-- List issues that this PR would close above. Ex: This PR closes #1, #2, #3. -->

## Link to prove your SVG is correct and up-to-date.
<!-- Link to an official page/wiki goes here. Anything that proves your SVGs are the correct ones. -->

https://github.com/helix-editor/helix
https://helix-editor.com/
https://github.com/helix-editor/helix/discussions/3754

You can see the original logo.svg in the github repo and on the website.
I chose #997ff8 as main color since it's on the logo, and purple variations are predominent on their website, the default theme in the helix editor itself, as well as their branding discussion.
I'm new to programming and github so everything feels challenging but the documentation was very helpful.

canaleal committed:
Merge pull request #2488 from devicons/develop

Release v2.17.0

Shyynek committed:
new icon: helix (original, plain, line)

Shyynek committed:
Merge branch 'develop' into add-helix-icons

weh reviewed (changes requested):
looks pretty good, but the viewbox is not 128x128. Also i am not sure if all the xml attributes are needed. Please fix the viewbox :)

weh commented:
you are right, the viewBox size is indeed 128x128, but it has a offset of 663.38 and 37.570 
i had this just recently so i can tell you how to fix it. 

First thing i would do, is to make sure all elements are in a group. This makes it easier to move all elements together.
so `ctrl + A` and `ctrl + G` should do that.

Next step is to set the X and Y offset of the viewbox to 0.
This will move the logo out of the visible viewbox, and you will have to zoom out to see it.

Then you can follow the Steps 5 and 6 in this tutorial
- https://github.com/devicons/devicon/wiki/Creating-original-Icons#step-5-center-the-group
- https://github.com/devicons/devicon/wiki/Creating-original-Icons#step-6-ungroup-all-paths

There are some Tutorials in the Wiki, and you can always ask if you need help.
I think there is also a list of SVOMG options that should be enabled/disabled to optimize the images.

Shyynek committed:
Rectified viewbox and attributes

Shyynek committed:
Merge branch 'add-helix-icons' of https://github.com/Shyynek/devicon into add-helix-icons
Rectified viewbox and html attributes

Shyynek commented:
@weh 

Thanks again for the help, following your hints i went back to the drawing board and fixed the icons viewboxes and htlm attributes (i deleted the attributes i knew wouldnt break the file, im learning on the fly tho)
It should be better now !

Shyynek committed:
Merge branch 'develop' into add-helix-icons

Shyynek committed:
Rectified helix icons names

Shyynek committed:
Merge branch 'add-helix-icons' of https://github.com/Shyynek/devicon into add-helix-icons
Pulling before pushing rectified helix icons names

weh committed:
omptimize svg

weh committed:
optimize svg

weh committed:
optimize svg

weh reviewed (approved):
nice work, i just updated the files to be on one line.

canaleal committed:
Merge branch 'develop' into add-helix-icons
